yt_dlp/extractor/nitter.py

```diff
@@ -68,96 +105,122 @@ class NitterIE(InfoExtractor):
                 'uploader': 'Mozilla',
                 'uploader_id': 'mozilla',
-                'uploader_url': 'https://%s/mozilla' % current_instance,
+                'uploader_url': f'https://{current_instance}/mozilla',
                 'upload_date': '20201027',
-                'timestamp': 1603820982
+                'timestamp': 1603820940,
+                'view_count': int,
+                'like_count': int,
+                'repost_count': int,
+                'comment_count': int,
             },
+            'expected_warnings': ['Ignoring subtitle tracks found in the HLS manifest'],
         }, {  # not the first tweet but main-tweet
-            'url': 'https://%s/TheNaturalNu/status/1379050895539724290#m' % current_instance,
+            'url': f'https://{current_instance}/firefox/status/1354848277481414657#m',
             'info_dict': {
-                'id': '1379050895539724290',
+                'id': '1354848277481414657',
                 'ext': 'mp4',
-                'title': 'Dorothy Zbornak - This had me hollering!!',
-                'description': 'This had me hollering!!',
+                'title': 'md5:bef647f03bd1c6b15b687ea70dfc9700',
+                'description': 'md5:5efba25e2f9dac85ebcd21160cb4341f',
                 'thumbnail': r're:^https?://.*\.jpg$',
-                'uploader': 'Dorothy Zbornak',
-                'uploader_id': 'TheNaturalNu',
-                'uploader_url': 'https://%s/TheNaturalNu' % current_instance,
-                'timestamp': 1617626329,
-                'upload_date': '20210405'
+                'uploader': 'Firefox 🔥',
+                'uploader_id': 'firefox',
+                'uploader_url': f'https://{current_instance}/firefox',
+                'upload_date': '20210128',
+                'timestamp': 1611855960,
+                'view_count': int,
+                'like_count': int,
+                'repost_count': int,
+                'comment_count': int,
             }
         }
     ]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
         parsed_url = compat_urlparse.urlparse(url)
-        base_url = '%s://%s' % (parsed_url.scheme, parsed_url.netloc)
+        base_url = f'{parsed_url.scheme}://{parsed_url.netloc}'
 
         self._set_cookie(parsed_url.netloc, 'hlsPlayback', 'on')
         full_webpage = self._download_webpage(url, video_id)
 
+        webpage = full_webpage
         main_tweet_start = full_webpage.find('class="main-tweet"')
         if main_tweet_start > 0:
             webpage = full_webpage[main_tweet_start:]
-        if not webpage:
-            webpage = full_webpage
 
-        video_url = '%s%s' % (base_url, self._html_search_regex(r'(?:<video[^>]+data-url|<source[^>]+src)="([^"]+)"', webpage, 'video url'))
+        video_url = '{}{}'.format(base_url, self._html_search_regex(
+            r'(?:<video[^>]+data-url|<source[^>]+src)="([^"]+)"', webpage, 'video url'))
```

Comment:
```suggestion
        video_url = '%s%s' % (base_url, self._html_search_regex(
            r'(?:<video[^>]+data-url|<source[^>]+src)="([^"]+)"', webpage, 'video url'))
```
% formatting is fine